Sources/AEPOptimize/Optimize+PublicAPI.swift

```diff
@@ -53,8 +55,22 @@ public extension Optimize {
                           type: EventType.optimize,
                           source: EventSource.requestContent,
                           data: eventData)
-
-        MobileCore.dispatch(event: event)
+        MobileCore.dispatch(event: event, timeout: 10) { responseEvent in
+            guard let responseEvent = responseEvent else {
+                let timeoutError = AEPOptimizeError(
+                    type: nil,
+                    status: 408,
+                    title: "Request Timeout",
+                    detail: "Update proposition request resulted in a timeout.",
```

Comment:
Can we add these in `OptimizeConstants`?

Reply:
Done